src/components/app.vue

```diff
@@ -71,11 +71,12 @@ export default {
           document.documentElement.style.backgroundColor = 'var(--color-accent-secondary)';
       });
 
+    const standalone = computed(() => route.meta.standalone);
```

Comment:
strangely UI doesn't change on route.meta.standalone changes :(

Reply:
Oh wait, are you saying that this computed ref works, but `$route.meta.standalone` didn't? Also very strange! This computed ref looks nice though.